birth-week-day-page/birth-week-day-page.js

```diff
@@ -0,0 +1,12 @@
+function calculateDay() {
+    const birthdate = new Date(document.getElementById('birthdate').value);
+    const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+
+    const a = 14 - Math.floor(birthdate.getUTCMonth() / 12);
```

Comment:
Algorithm requires  `(14 - month) / 12`
But in your case it is `14 - (month / 12)`